File: crates/codededup/tests/fixtures/csharp-small/Alpha.cs
namespace Alpha
{
    public class Processor
    {
        public int Compute(int input)
        {
            if (input < 0)
            {
                return 0;
            }
            int total = 0;
            for (int index = 0; index < input; index = index + 1)
            {
                total = total + index;
            }
            return total;
        }
    }
}

File: crates/deslop/tests/fixtures/php-small/alpha.php
<?php

function compute(int $input): int
{
    if ($input < 0) {
        return 0;
    }
    $total = 0;
    for ($index = 0; $index < $input; $index++) {
        $total = $total + $index;
    }
    return $total;
}
